### DISCUSSION TITLE
editor: metronome in lazer ticks only after one beat

### DISCUSSION THREAD

**Discussion by Dopaminos** [ppy/osu#21494] [Q&A]
in lazer, metronome starts to tick only after beat after selected will end,
stable metronome works instantly. resolving this issue would be helpful in timing cause i don't really want to use stable editor without waveform to do that ,_,

https://user-images.githubusercontent.com/42616718/205231135-0e704de1-1baf-4068-bfa7-6c6fe8b91fec.mp4

**Comment by peppy** [comment]
Potentially related to  #21988
